docs/tutorial/workarounds.md

```diff
@@ -0,0 +1,142 @@
+# Common tricks used in (T)FHE
+
+As explained in the [Basics of FHE](../getting-started/fhe_basics.md), the challenge for developers
+is to adapt their code to fit FHE constraints. In this document we have collected some common examples
+to illustrate the kind of optimization one can do to get better performance.
+
+{% hint style="info" %}
+All code snippets provided here are for educational purpose. In future version of Concrete, some
```

Comment:
I find "All code snippets provided here are for educational purpose. " a bit strange. It sound like the site where they say "use this code at your own peril". I would remove

Reply:
it's workarounds until functions are properly supported, that's why I thought a disclaimer could be nice.

Reply:
ok